pages/js-templates/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Javascript Page Generator
+ */
+
+/* eslint strict: ["off"] */
+
+'use strict';
+
+import componentExists from '../../utils/componentExists.js'
+import config from '../../constants.js'
+
+export default {
+  description: 'Add a page for your application',
```

Comment:
@khus29 right now there is too much duplication in generator code of TS and JS. Why is that? Ideally only templates should have been different. 

Since generator is very much doing exactly the same thing. few differences like file extensions and copy paths can be conditional.

Reply:
Yes Vinod, there is lot of duplication right now, while fixing TS/JS error I have separated them. However, refactoring is in plan after monorepo implementation. We can hold this PR until then.